floodlight/core/pitch.py

```diff
@@ -147,3 +153,967 @@ def center(self):
+            An axes to which all elements of the sport specific pitch are added.
+        """
+
+        # customizing visualization
+        if color_scheme == "black_white":
+            ax.set_facecolor("white")
+            outline = "black"
+            innerline = "black"
+            fill = "white"
+            goal = "black"
+        else:
+            ax.set_facecolor("skyblue")
+            outline = "white"
+            innerline = "white"
+            fill = "khaki"
+            goal = "white"
+
+        # key data for the handball pitch
+        xmin, xmax = self.xlim
+        ymin, ymax = self.ylim
+
+        x_range = xmax - xmin
+        y_range = ymax - ymin
+        x_half = (xmax + xmin) / 2
+        y_half = (ymax + ymin) / 2
+
+        linewidth = kwargs.get("linewidth", 1)
+
+        # points on the field which are used for plotting all lines and arcs
+        left_side_right_goal_post = (xmin, ymin + y_range * 0.425)
+        left_side_left_goal_post = (xmin, ymin + y_range * 0.575)
+        right_side_left_goal_post = (xmax, ymin + y_range * 0.425)
+        right_side_right_goal_post = (xmax, ymin + y_range * 0.575)
+        width_hight_free_throw_line = y_range * 0.9
+        width_hight_goal_area_line = y_range * 0.6
+        radius_goal_area_line_arc = y_range * 0.3
+        goal_width = y_range * 0.15
+        goal_height = y_range * 0.1
+        y_center_to_post = goal_width / 2
+        y_value_goal_lower_post = y_half - y_center_to_post
+        y_value_goal_upper_post = y_half + y_center_to_post
+        y_pos_lower_goal_posts = ymin + y_range * 0.425
+        y_pos_upper_goal_posts = ymin + y_range * 0.575
+        left_side_x_pos_goal_area_line_edge = xmin + (x_range * 0.15)
+        right_side_x_pos_goal_area_line_edge = xmax - (x_range * 0.15)
+        left_side_x_pos_free_throw_line_edge = xmin + (x_range * 0.225)
+        right_side_x_pos_free_throw_line_edge = xmax - (x_range * 0.225)
+        left_side_4m_line_x_pos = xmin + (x_range * 0.1)
+        lower_y_pos_4m_line = ymin + (y_range * 0.49625)
+        upper_y_pos_4m_line = ymin + (y_range * 0.50375)
+        left_side_7m_line_x_pos = xmin + (x_range * 0.175)
+        right_side_7m_line_x_pos = xmax - (x_range * 0.175)
+        lower_y_pos_7m_line = ymin + (y_range * 0.475)
+        upper_y_pos_7m_line = ymin + (y_range * 0.525)
+        x_margin = x_range / 40
+        y_margin = y_range / 20
+
+        # set up the boundaries of ax
+        ax.set_xlim(
+            [
+                xmin - (goal_height + x_margin),
+                xmax + (goal_height + x_margin),
+            ]
+        )
+        ax.set_ylim([ymin - y_margin, ymax + y_margin])
+
+        # paint handball pitch with all properties
+        # main boundaries
+        ax.plot(
+            [xmin, xmin],
+            [ymin, ymax],
+            color=outline,
+            scalex=False,
+            scaley=False,
+            linewidth=linewidth,
+            zorder=0,
```

Comment:
imo, maybe ``zorder`` should be a **kwarg. just in case anybody would want to fiddle with the order themself?

Reply:
Alright I would implement it like:

zorder = kwargs.get('zorder') if kwargs.get('zorder') else 0

Btw, I was wondering if a kwarg is not written properly or if a kwarg doesn't exist in plt.subplot(**kwargs) and other Matplotlib elements should we make a try and catch/exception block to print an error message or should we just leave that to matplotlib?